From kernel-janitors-bounces@lists.osdl.org Wed Feb 15 13:45:49 2006
From: Matthew Martin <lihnucks@gmail.com>
To: kernel-janitors@lists.osdl.org
Date: Wed, 15 Feb 2006 15:41:25 -0600
Message-Id: <1140039685.14352.1.camel@localhost.localdomain>
Subject: USB: Fix warning in drivers/usb/media/ov511.c

Gcc 4.0.2 had the warning:

drivers/usb/media/ov511.c: In function 'show_exposure':
drivers/usb/media/ov511.c:5642: warning: 'exp' may be used uninitialized
in this function

Here is the patch to fix that warning.

Signed-off-by: Matthew Martin <lihnucks@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/media/ov511.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/media/ov511.c
+++ gregkh-2.6/drivers/usb/media/ov511.c
@@ -5639,7 +5639,7 @@ static CLASS_DEVICE_ATTR(hue, S_IRUGO, s
 static ssize_t show_exposure(struct class_device *cd, char *buf)
 {
 	struct usb_ov511 *ov = cd_to_ov(cd);
-	unsigned char exp;
+	unsigned char exp = 0;
 
 	if (!ov->dev)
 		return -ENODEV;
